services/simcore/.env

```diff
@@ -6,56 +6,60 @@
 BF_API_KEY=none
 BF_API_SECRET=none
 
-CATALOG_HOST=${PREFIX_STACK_NAME}_catalog
+CATALOG_HOST=${SIMCORE_STACK_NAME}_${PREFIX_STACK_NAME}_catalog
 
-DIRECTOR_HOST=${PREFIX_STACK_NAME}_director
+DIRECTOR_HOST=${SIMCORE_STACK_NAME}_${PREFIX_STACK_NAME}_director
 
-DOCKER_IMAGE_TAG=staging-latest
+DOCKER_IMAGE_TAG=master-latest
 
 EXTRA_HOSTS_SUFFIX=osparc.local
 
 OPS_MONITORED_NETWORK=monitored_network
 OPS_PUBLIC_NETWORK=public-network
 
 POSTGRES_DB=simcoredb
-POSTGRES_ENDPOINT=${PREFIX_STACK_NAME}_postgres:5432
-POSTGRES_HOST=${PREFIX_STACK_NAME}_postgres
-
+POSTGRES_ENDPOINT=local_postgres:5432
+POSTGRES_HOST=local_postgres
 POSTGRES_PASSWORD=adminadmin
 POSTGRES_PORT=5432
-POSTGRES_USER=scu
+POSTGRES_USER=admin
 
-PUBLISHED_HOST_NAME=staging.osparc.local
+PUBLISHED_HOST_NAME=osparc.local
 MONITORING_DOMAIN=monitoring.osparc.local
 
 RABBIT_PORT=5672
-RABBIT_HOST=${PREFIX_STACK_NAME}_rabbit
+RABBIT_HOST=local_rabbit
 RABBIT_LOG_CHANNEL=comp.backend.channels.log
 RABBIT_PROGRESS_CHANNEL=comp.backend.channels.progress
-RABBIT_USER=simcore
-RABBIT_PASSWORD=simcore
+RABBIT_USER=admin
+RABBIT_PASSWORD=adminadmin
 
-REDIS_HOST=${PREFIX_STACK_NAME}_redis
+REDIS_HOST=local_redis
 REDIS_PORT=6379
 
 REGISTRY_AUTH=True
 REGISTRY_PW=adminadmin
 REGISTRY_SSL=True
-REGISTRY_URL=masu.speag.com
+REGISTRY_URL=registry.osparc.local
 REGISTRY_USER=admin
 
 S3_ACCESS_KEY=adminadmin
-S3_BUCKET_NAME=${PREFIX_STACK_NAME}-simcore
-S3_ENDPOINT=osparc.local:10000
+S3_BUCKET_NAME=local-simcore
+S3_ENDPOINT=storage.osparc.local
 S3_SECRET_KEY=adminadmin
 S3_SECURE=1
 
 SIMCORE_COMPUTATIONAL_NETWORK_SUBNET=172.11.0.0/16
 SIMCORE_INTERACTIVE_NETWORK_SUBNET=172.10.0.0/16
 SIMCORE_PORT=9081
-
-STORAGE_ENDPOINT=${PREFIX_STACK_NAME}_storage:8080
-STORAGE_HOST=${PREFIX_STACK_NAME}_storage
+SMTP_HOST=smtp.osparc.local
+SMTP_PORT=25
+SMTP_USERNAME=admin
+SMTP_PASSWORD=adminadmin
+SMTP_TLS_ENABLED=1
+
+STORAGE_ENDPOINT=${SIMCORE_STACK_NAME}_${PREFIX_STACK_NAME}_storage:8080
+STORAGE_HOST=${SIMCORE_STACK_NAME}_${PREFIX_STACK_NAME}_storage
```

Comment:
this looks weird... are you sure it should not be just ${PREFIX_STACK_NAME}_storage ???